src/measurement/store.ts

```diff
@@ -73,14 +71,17 @@ export class MeasurementStore {
 			results,
 		};
 		const measurementWithoutDefaults = this.removeDefaults(measurement, request);
+		const testsToProbes = Object.fromEntries(Array.from(onlineProbesMap, ([ testId, probe ]) => [ `${id}_${testId}`, probe.uuid ]));
 
 		await Promise.all([
 			this.redis.hSet('gp:in-progress', id, startTime.getTime()),
-			this.redis.set(getMeasurementKey(id, 'probes_awaiting'), onlineProbesMap.size, { EX: probesAwaitingTtl }),
+			this.redis.set(getMeasurementKey(id, 'probes_awaiting'), onlineProbesMap.size, { EX: config.get<number>('measurement.timeout') + 30 }),
```

Comment:
Any particular reason we're increasing this as well?

Reply:
Just for the value to be the same, as there is no reason for delay to be `+5` in one case and `+30` in another.